app/controllers/administration_controller.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class AdministrationController < ApplicationController
+  # binding.pry
   before_action :authenticate_admin!
   def index; end
```

Comment:
je pense qu'on peut virer def index; end

Reply:
No, it tells us that we have an index view.

Reply:
yep agree with @Crysicia